src/components/shared/pcsGrid/pcsGridConfig.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+/* This file contains default values useful for creating PcsGrid */
+
+/** The default value for PcsGrid cells that are empty */
+export const EMPTY_FIELD_VAL = '---';
+
+/** The default formatting for dates in the PcsGrid */
+export const DEFAULT_TIME_FORMAT = 'hh:mm:ss MM.DD.YYYY';
```

Comment:
How are we overriding this format?

Reply:
Currently, it isn't being overridden. This should be replaced later by a component that handles rendering timestamps.